### DISCUSSION TITLE
Vant 官方提供了一份默认的省市区数据，可以通过 @vant/area-data 引入。

### DISCUSSION THREAD

**Discussion by gitYeC** [youzan/vant#9326] [Q&A]
npm安装完了，通过import { areaList } from '@vant/area-data';引用报错。@vant/area-data.js" is not defined

**Comment by wwwkj** [comment]
https://blog.csdn.net/u014678583/article/details/104004693/?utm_medium=distribute.pc_relevant.none-task-blog-2~default~baidujs_title~default-0.no_search_link&spm=1001.2101.3001.4242.1 看看这个是不是能帮到你

**Comment by gitYeC** [comment]
已经弄好了，感谢大哥




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "youzan/vant"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年10月27日(星期三) 下午2:42
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [youzan/vant] Vant 官方提供了一份默认的省市区数据，可以通过 @vant/area-data 引入。 (Discussion #9326)





 
https://blog.csdn.net/u014678583/article/details/104004693/?utm_medium=distribute.pc_relevant.none-task-blog-2~default~baidujs_title~default-0.no_search_link&amp;spm=1001.2101.3001.4242.1 看看这个是不是能帮到你
 
—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
